stores/inmemory/store.go

```diff
@@ -26,7 +26,7 @@ type InMemStore struct {
 	highestAttestation     map[string]*phase0.AttestationData
 
 	highestProposalLock sync.RWMutex
-	highestProposal     map[string]phase0.Slot
+	highestProposal     map[string]uint64
```

Comment:
why is this changed?

Reply:
> as written in description of the PR

Replaced phase0.Slot with uint64 due to encoding changes in go-eth2-client (they changed encoding from number to string, so we use uint64 to preserve old behavior)